src/index.js

```diff
@@ -133,3 +126,171 @@ export class WealthscopeSdk {
     }
   }
 }
+
+export class WealthscopeApiClient {
```

Comment:
can we put `WealthscopeApiClient` and `WealthscopeSdk` into their own separate files?
We can then import them here, and export them in a single object.